src/effects/LUT.tsx

```diff
@@ -1,19 +1,24 @@
-import { LUTEffect } from 'postprocessing'
-import React, { forwardRef, Ref, useMemo, useLayoutEffect } from 'react'
-import { Texture } from 'three'
 import { useThree } from '@react-three/fiber'
+import { LUT3DEffect, BlendFunction } from 'postprocessing'
+import React, { forwardRef, Ref, useLayoutEffect, useMemo } from 'react'
+import { Texture } from 'three'
 
-type LUTProps = ConstructorParameters<typeof LUTEffect>[1] & {
+type LUTProps = {
   lut: Texture
+  blendFunction?: BlendFunction
+  tetrahedralInterpolation?: boolean
 }
```

Comment:
We'll want to go through and export props for all components wherever possible or appropriate.

Reply:
:+1: